src/App.js

```diff
@@ -1,13 +1,7 @@
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Welcome to Math Magician App</h1>
-      </header>
-    </div>
-  );
+  return <Calculator />;
 }
```

Comment:
- [ ] Kindly make sure the `App.js` main component file s a **class-based component** and not a **function-based component** as required for the project. Make sure you update it to a  **[class-based component](https://www.w3schools.com/react/react_class.asp)**

Reply:
Hi @tresorsawasawa , I think this is not in the feature branch but I understood the issue and I fixed it.